entrypoint.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
 
+su - www-data << 'EOF'
+    composer install --no-dev --no-interaction
+
+    npm install
+    npm run build
+
+    cp .env.docker .env
+
+    php artisan storage:link
+EOF
```

Comment:
What's up with this? We're already executing operations like these in `Dockerfile` (see line 41 till 48).

Reply:
That is true, but this way the Dockerfile can be used for production and local development. Because when building it we want to run it, but it should also run whenever the Dockerfile is ran from the docker compose file.